README.md

```diff
@@ -14,7 +14,9 @@
 
 1. Run `npm install -g urcli`.
 1. Get a new token from the API Access link in the [Reviewer Dashboard](https://review.udacity.com/#!/submissions/dashboard).
-<img src="http://i.imgur.com/QH7onbk.png" alt="Token retrieval" width="500px">
+
+    <img src="http://i.imgur.com/QH7onbk.png" alt="Token retrieval" width="500px">
```

Comment:
I don't know why we would want to conform to commonMark or where it doesn't display the image correctly. Can you elaborate in an issue?